### ISSUE TITLE
hash generation function does not produce unique values

### ISSUE THREAD

**Issue by khannedy** [gphotosuploader/gphotos-uploader-cli#299]
after push, and input the authorization code, always get this error

```
Enter the passphrase to open the token store:
panic: crypto/hmac: hash generation function does not produce unique values

goroutine 1 [running]:
crypto/hmac.New(0xc00014d168, 0xc7f248, 0x0, 0x0, 0xc0002c2000, 0x7f1446dc2ac0)
	/usr/local/go/src/crypto/hmac/hmac.go:143 +0x370
github.com/dvsekhvalnov/jose2go/kdf.DerivePBKDF2(0xc7f248, 0x0, 0x0, 0xc00002c420, 0x1f, 0x30, 0x2000, 0x80, 0x9c9a98, 0xc0002c2000, ...)
	/root/golang/work/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/kdf/pbkdf2.go:15 +0xad
github.com/dvsekhvalnov/jose2go.(*Pbse2HmacAesKW).WrapNewKey(0xc0000b00a8, 0x100, 0x887180, 0xc80920, 0xc0001b6540, 0x203001, 0x40, 0x40, 0x3, 0x33000000004843bb, ...)
	/root/golang/work/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/pbse2_hmac_aeskw.go:50 +0x439
github.com/dvsekhvalnov/jose2go.encrypt(0xc0002c0500, 0x275, 0x280, 0xc0001b6540, 0x887180, 0xc80920, 0xc0002c0775, 0xb, 0x200, 0xc0002c0500)
	/root/golang/work/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/jose.go:305 +0x20a
github.com/dvsekhvalnov/jose2go.EncryptBytes(0xc0002c0500, 0x275, 0x280, 0x933667, 0x12, 0x92f0ac, 0x7, 0x887180, 0xc80920, 0xc00014d680, ...)
	/root/golang/work/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/jose.go:222 +0x32f
github.com/dvsekhvalnov/jose2go.Encrypt(...)
	/root/golang/work/pkg/mod/github.com/dvsekhvalnov/jose2go@v0.0.0-20180829124132-7f401d37b68a/jose.go:187
github.com/99designs/keyring.(*fileKeyring).Set(0xc00012b290, 0x934491, 0x13, 0xc0001fe300, 0x170, 0x180, 0x0, 0x0, 0x0, 0x0, ...)
	/root/golang/work/pkg/mod/github.com/99designs/keyring@v1.1.5/file.go:144 +0x2d9
github.com/gphotosuploader/gphotos-uploader-cli/internal/datastore/tokenstore.(*KeyringRepository).Set(0xc0000938c0, 0x934491, 0x13, 0xc00006c840, 0xc0002b6000, 0x67)
	/root/golang/work/pkg/mod/github.com/gphotosuploader/gphotos-uploader-cli@v1.2.0/internal/datastore/tokenstore/keyring.go:66 +0x118
github.com/gphotosuploader/gphotos-uploader-cli/internal/datastore/tokenstore.(*TokenManager).StoreToken(0xc0000938d0, 0x934491, 0x13, 0xc00006c840, 0x1, 0x1)
	/root/golang/work/pkg/mod/github.com/gphotosuploader/gphotos-uploader-cli@v1.2.0/internal/datastore/tokenstore/tokenmanager.go:43 +0xa7
github.com/gphotosuploader/gphotos-uploader-cli/internal/app.(*App).NewOAuth2Client(0xc0000d65c0, 0x9c8dd0, 0xc00009e010, 0xc00012e050, 0x48, 0xc0000ca2b8, 0x18, 0x9404ac, 0x29, 0x93d397, ...)
	/root/golang/work/pkg/mod/github.com/gphotosuploader/gphotos-uploader-cli@v1.2.0/internal/app/oauth.go:58 +0x252
github.com/gphotosuploader/gphotos-uploader-cli/internal/cmd.(*PushCmd).Run(0xc0000b01f8, 0xc0000b7180, 0xc7f248, 0x0, 0x0, 0x0, 0x0)
	/root/golang/work/pkg/mod/github.com/gphotosuploader/gphotos-uploader-cli@v1.2.0/internal/cmd/push.go:100 +0xb10
github.com/spf13/cobra.(*Command).execute(0xc0000b7180, 0xc7f248, 0x0, 0x0, 0xc0000b7180, 0xc7f248)
	/root/golang/work/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc4a260, 0x200000003, 0xc000000180, 0xc000109f78)
	/root/golang/work/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x30b
github.com/spf13/cobra.(*Command).Execute(...)
	/root/golang/work/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/gphotosuploader/gphotos-uploader-cli/internal/cmd.Execute()
	/root/golang/work/pkg/mod/github.com/gphotosuploader/gphotos-uploader-cli@v1.2.0/internal/cmd/root.go:48 +0x31
main.main()
	/root/golang/work/pkg/mod/github.com/gphotosuploader/gphotos-uploader-cli@v1.2.0/main.go:10 +0x25
```

**Comment by pacoorozco** [comment]
This bug seems to be duplicated. It was fixed on #294 and released on version 3.1.0

Are you using the latest version of this CLI? Can you run it using debug mode?

**Comment by khannedy** [comment]
update to latest, it's fix the problem
